### PR TITLE
Changes to the container apps UI

### PR DESCRIPTION
**Pull request by nturinski**
Some changes to the UI:

OLD MULTIPLE:
image
NEW MULTIPLE: (I want to use this icon for Revision management)
image
image

OLD SINGLE:
image

NEW SINGLE:
image

### CODE REVIEW

File: src/tree/ContainerAppItem.ts [hunk 1]
```diff
@@ -60,23 +59,22 @@ export class ContainerAppItem implements ContainerAppsItem {
     portalUrl: Uri = createPortalUrl(this.subscription, this.containerApp.id);
 
     async getChildren(): Promise<TreeElementBase[]> {
+        // this should have configuration and revisions
```
Comment (MicroFish91) [inline src/tree/ContainerAppItem.ts:62]: Was this like a todo comment?  I see the configurations and revisions below already.

File: src/tree/ContainerAppItem.ts [hunk 2]
```diff
@@ -60,23 +59,22 @@ export class ContainerAppItem implements ContainerAppsItem {
...
+
         const result = await callWithTelemetryAndErrorHandling('getChildren', async (context) => {
             const children: TreeElementBase[] = [];
-            children.push(this.containerApp.configuration?.dapr?.enabled ? new DaprEnabledItem(this.containerApp, this.containerApp.configuration.dapr) : createDaprDisabledItem(this.containerApp));
-
             const client: ContainerAppsAPIClient = await createContainerAppsAPIClient([context, createSubscriptionContext(this.subscription)]);
-            const revisionData = await client.containerAppsRevisions.getRevision(this.resourceGroup, this.name, nonNullProp(this.containerApp, 'latestRevisionName'));
+
```
Comment (MicroFish91) [inline src/tree/ContainerAppItem.ts:67]: Nit: extra new line

File: src/tree/RevisionItem.ts [hunk 3]
```diff
@@ -33,11 +36,14 @@ export class RevisionItem implements RevisionsItemModel {
...
-            localize('inactive', 'Inactive') :
-            this.revision.name === this.containerApp.latestRevisionName ?
-                localize('latest', 'Latest') :
-                undefined;
+        const description = this.revisionsMode === KnownActiveRevisionsMode.Multiple ?
+            !this.revision.active ?
+                localize('inactive', 'Inactive') :
+                this.revision.name === this.containerApp.latestRevisionName ?
+                    localize('latest', 'Latest') :
+                    undefined :
```
Comment (MicroFish91) [inline src/tree/RevisionItem.ts:44]: The number of ternaries here:

image

😂

**Review by MicroFish91 — APPROVED** [review]
Looks good!

File: src/tree/ContainerAppItem.ts [hunk 4]
```diff
@@ -103,7 +101,14 @@ export class ContainerAppItem implements ContainerAppsItem {
     }
 
     static async Get(context: IActionContext, subscription: AzureSubscription, resourceGroupName: string, containerAppName: string): Promise<ContainerAppModel> {
+        const credential = createSubscriptionContext(subscription).credentials;
         const client: ContainerAppsAPIClient = await createContainerAppsClient(context, subscription);
+        const gClient = await createGenericClient(context, createSubscriptionContext(subscription));
```
Comment (MicroFish91) [inline src/tree/ContainerAppItem.ts:106]: I actually missed this on the first pass, but why is this gClient used here?

File: src/tree/ContainerAppItem.ts (hunk 1 above)
Comment (nturinski) [reply src/tree/ContainerAppItem.ts:62]: Yeah haha, notes while I am thinking of the UI. Whoops 😳

File: src/tree/ContainerAppItem.ts (hunk 4 above)
Comment (nturinski) [reply src/tree/ContainerAppItem.ts:106]: Huh yeah, this is weird. I must have added this to test something and forgot to remove it...

**Comment by nturinski** [comment]
Note: Still need to change the icon for multiple. Also, we should change the label for the revision name in single. I'm thinking of maybe just putting "Latest" there or something, but will think on this more.

**Pull request merged**